rma/security/ir.model.access.csv

```diff
@@ -2,6 +2,7 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_rma_team_user_own,rma.team.user.own,model_rma_team,rma_group_user_own,1,0,0,0
 access_rma_team_manager,rma.team.manager,model_rma_team,rma_group_manager,1,1,1,1
 access_rma_portal,rma.portal,model_rma,base.group_portal,1,0,0,0
+access_rma_internal_user,rma.internal_user,model_rma,base.group_user,1,0,0,0
```

Comment:
This permission shouldn't exist.

Reply:
Well, is to avoid patching with `sudo()` every possible method reading an `rma` related field. It's harmless, as it's needed the proper record rule to access to any information.

Reply:
I prefer to not give that one, as people can modify record rules, but not ACLs, so you can find that someone without RMA permissions is able to see some of them after any modification of the record rules, thinking that only affects RMA users.

Reply:
Ok, changes done.